### ISSUE TITLE
Can't connect on Linux with RC2 - Socket error

### ISSUE THREAD

**Issue by SonicGD** [bchavez/RethinkDb.Driver#63]
Hello, me again =) v2.3.3-beta-1 build 533 works great with RC2 on Windows. But when i deploy it on Linux - new error shows up:

```
Sockets on this platform are invalid for use after a failed connection attempt, and as a result do not support attempts to connect to multiple endpoints.
```

connection code:
```cs
_conn = RDB.Connection().Hostname(_host).Port(_port).Connect();
```

I believe it caused by this commit - https://github.com/dotnet/corefx/commit/30bd4b79185d83f269f6a6e432881998a0158178

**Comment by bchavez** [comment]
Еще раз здравствуйте =) Спасибо. :ru:

Fixed in [`2.3.4-beta-1`](https://www.nuget.org/packages/RethinkDb.Driver/2.3.4-beta-1) net core. Please let me know how it goes.

:art: :dash: [**Marshmello - CoLoUR**](https://www.youtube.com/watch?v=6UhNJC8OyVA)

**Comment by SonicGD** [comment]
Seems good. Спасибо =)
